src/plotSearch/components/plotSearchSections/basicInfo/PlotSearchSiteEditCustomDetailedPlan.js

```diff
@@ -0,0 +1,280 @@
+            <FormTextTitle>
+              {'Kaavayksikön käyttötarkoitus'}
+            </FormTextTitle>
+            <FormText>{currentCustomDetailedPlan && getLabelOfOption(customDetailedPlanIntendedUseOptions, currentCustomDetailedPlan.intended_use.id) || '-'}</FormText>
+          </Column>
+          <Column small={6} medium={4} large={3}>
+            <FormTextTitle>
+              {'Kokonaisala'}
+            </FormTextTitle>
+            <FormText>
+              {`${get(currentCustomDetailedPlan, 'area')} m²` || '-'}
+            </FormText>
+          </Column>
+          <Column small={6} medium={4} large={3}>
+            <FormTextTitle>
+              {'Osoite'}
+            </FormTextTitle>
+            <FormText>
+              {get(currentCustomDetailedPlan, 'address') || '-'}
+            </FormText>
+          </Column>
+          <Column small={6} medium={4} large={3}>
+            <FormTextTitle>
+              {'Asemakaava'}
+            </FormTextTitle>
+            {get(currentCustomDetailedPlan, 'detailed_plan')
+              ? <ExternalLink
+                href={createPaikkatietovipunenUrl(`helreport/planpdfloader/?id=${get(currentCustomDetailedPlan, 'detailed_plan')}`)}
+                text={get(currentCustomDetailedPlan, 'detailed_plan')}
+              />
+              : <FormText>-</FormText>
+            }
+          </Column>
+          <Column small={6} medium={4} large={3}>
+            <FormTextTitle>
+              {'Asemakaavan viimeisin käsittelypvm'}
+            </FormTextTitle>
+            <FormText>
+              {formatDate(get(currentCustomDetailedPlan, 'detailed_plan_latest_processing_date')) || '-'}
+            </FormText>
+          </Column>
+          <Column small={6} medium={4} large={3}>
+            <FormTextTitle>
+              {'Asemakaavan viimeisin käsittelypvm. selite'}
+            </FormTextTitle>
+            <FormText>
+              {get(currentCustomDetailedPlan, 'detailed_plan_latest_processing_date_note') || '-'}
+            </FormText>
+          </Column>
+          <Column small={6} medium={4} large={3}>
+            <FormTextTitle>
+              {'Kaavayksikön olotila'}
+            </FormTextTitle>
+            <FormText>{currentCustomDetailedPlan && getLabelOfOption(customDetailedPlanStateOptions, currentCustomDetailedPlan.state.id) || '-'}</FormText>
+          </Column>
+          <Column small={6} medium={4} large={3}>
+            <FormTextTitle>
+              {'Kaavayksikön laji'}
+            </FormTextTitle>
+            <FormText>{currentCustomDetailedPlan && getLabelOfOption(customDetailedPlanTypeOptions, currentCustomDetailedPlan.type.id) || '-'}</FormText>
+          </Column>
+          <Column small={6} medium={4} large={3}>
+            <FormTextTitle>
+              {'Kokonaisrakennusoikeus'}
+            </FormTextTitle>
+            <FormText>
+              {get(currentCustomDetailedPlan, 'rent_build_permission') || '-'}
```

Comment:
This is in square metres; the unit is missing.

Reply:
The unit was added but in a way that makes the default "no data" case never happen, as an empty field would still end up with a truthy string. It needs to be rearranged a bit, even if the flow in that case isn't as neat.